baseapp-payments/setup.cfg

```
@@ -0,0 +1,41 @@
+[metadata]
+name = baseapp_payments
+version = 0.14
+description = A BaseApp app to handle payments.
+long_description = file: README.md
+url = https://bitbucket.org/silverlogic/baseapp-payments-django
+author = The SilverLogic
+author_email = dev@tsl.io
+license = BSD-3-Clause  # Example license
+classifiers =
+    Environment :: Web Environment
+    Framework :: Django
+    Framework :: Django :: 2.2
```

Also adjust Django version here?
```suggestion
    Framework :: Django :: 3.2
```